src/pages/Homepage.js

```diff
@@ -0,0 +1,43 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import HeroComponent from '../components/HeroComponent';
+
+import { getCrypto } from '../redux/currencies/currenciesSlIce';
+import SingleCurrency from '../components/SIngleCurrency';
+
+const Homepage = () => {
+  const { curencies } = useSelector((state) => state.currencies);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getCrypto());
+  }, [dispatch]);
+
+  return (
+    <section className="section-container">
```

Comment:
- [ ] Kindly make sure to follow the provided design, you can use the `opacity` property to add a different color shad for every second column on the Home page or just add two different shades of the same color to style the coins cards

|Your app|Provided design|
|-|-|
|![Screenshot_20230322_163017](https://user-images.githubusercontent.com/25227022/226973187-5a87ce29-9282-4780-8ed9-5866b8d7e3cc.png)|![Screenshot_20230322_163141](https://user-images.githubusercontent.com/25227022/226973631-feb8a27e-2bbd-4286-9f56-002d91357c54.png)|